src/app/shared/services/popup.service.ts

```diff
@@ -10,24 +10,65 @@ export class PopupService {
     }
 
     new(options: SweetAlertOptions): Observable<SweetAlertResult> {
-        return from(swal(options));
+        return from(swal({...options, ...this.setDefaultOptions(options)}));
     }
 
     info(message: string, footer?: string): Observable<SweetAlertResult> {
         return from(swal({
             type: 'info',
             titleText: 'Info',
             text: message,
-            footer: footer
+            footer: footer,
+            customClass: 'popup-info',
+            position: 'top'
         }));
     }
 
     success(message: string, footer?: string): Observable<SweetAlertResult> {
         return from(swal({
-            type: 'info',
-            titleText: 'Success',
+            type: 'success',
+            titleText: 'Success!',
+            text: message,
+            footer: footer,
+            customClass: 'popup-success',
+            position: 'top',
+            confirmButtonText: 'Continue <i class="fas fa-arrow-right ml-3"></i>'
+        }));
+    }
+
+    warning(message: string, footer?: string): Observable<SweetAlertResult> {
+        return from(swal({
+            type: 'warning',
+            titleText: 'Warning!',
+            text: message,
+            footer: footer,
+            customClass: 'popup-warning',
+            position: 'top'
+        }));
+    }
+
+    error(message: string, footer?: string): Observable<SweetAlertResult> {
+        return from(swal({
+            type: 'error',
+            titleText: 'Error!',
             text: message,
-            footer: footer
+            footer: footer,
+            customClass: 'popup-error',
+            position: 'top'
         }));
     }
+
+    setDefaultOptions(options: SweetAlertOptions): SweetAlertOptions {
+        const defaults: SweetAlertOptions = {
+            position: 'top',
+            customClass: '',
+            confirmButtonText: 'Ok'
+        };
+        defaults.customClass = `popup-${options.type}`;
```

Comment:
One day when we introduce PurgeCSS to remove unused CSS classes from our application bundle, it won't recognize that `.popup-success` or `.popup-error` exist, so it will accidentally remove it. Let it stay like this, but in the future, try not to interpolate CSS classes. I don't have a valid solution for this situation, but we could encounter on this issue later, so we can remember it.

Reply:
Btw, it seems like this will fail if the user doesn't provide `options.type`. Did you ensure that this is mandatory?

Reply:
I could use if statements instead of interpolation to add classes to default options.
Well, the user shouldn't control how and where popups are displayed unless we are going to add where popups are displayed in app configuration?

Reply:
> I could use if statements instead of interpolation to add classes to default options.

If you have a way to do it (and if it doesn't look ugly), you can.

> Well, the user shouldn't control how and where popups are displayed unless we are going to add where popups are displayed in app configuration?

I don't understand what you want to say. 
We need to ensure if the user (we, developers, consumers of our API) uses `popupService.new()`, and doesn't provide a type, the application should not fail. I am pretty sure that info or success is the default option to Sweet Alert library, so it will fall back to this, but fail in our case in line 67 because the wrong `customClass` will be assigned. 

My suggestion is something like:
```
defaults.customClass = `popup-${options.type || 'success'}`;
```
Or, better one (considering issue with PurgeCSS mentioned above):
```
defaults.customClass = options.type ? `popup-${options.type}` : 'popup-success';
```